var/spack/repos/builtin/packages/py-atlas-building-tools/package.py

```diff
@@ -11,13 +11,14 @@ class PyAtlasBuildingTools(PythonPackage):
     homepage = "https://bbpcode.epfl.ch/browse/code/nse/atlas-building-tools/tree/"
     git      = "ssh://bbpcode.epfl.ch/nse/atlas-building-tools"
 
+    version('0.1.3', tag='atlas-building-tools-v0.1.3')
     version('0.1.2', tag='atlas-building-tools-v0.1.2')
     version('0.1.1', tag='atlas_building_tools-v0.1.1')
 
     depends_on('py-setuptools', type=('build', 'run'))
 
     depends_on('py-cgal-pybind@0.1.1:', type=('build', 'run'), when='@0.1.2:')
-    depends_on('py-cgal-pybind@0.1.0', type=('build', 'run'), when='@0.1.1')
+    depends_on('py-cgal-pybind@0.1.0:', type=('build', 'run'), when='@0.1.1')
```

Comment:
did this change suddenly?

Reply:
I took the opportunity of the release to make this change too. My assumption: it is recommended to not cap nor pin a version when it is possible. (The absence of ":" was a mistake).

Reply:
OK, thanks! CI passed, so this is good to be merged.